package-testing/python-sdk-relay/src/server.py

```diff
@@ -160,7 +160,8 @@ def initialize_client_and_wait():
     initialize_client_and_wait()
     
     port = int(environ.get('SDK_RELAY_PORT', 7001))
-    host = environ.get('SDK_RELAY_HOST', '0.0.0.0')
+    #host = environ.get('SDK_RELAY_HOST', '0.0.0.0')
+    host = '0.0.0.0'
```

Comment:
needed `0.0.0.0` but `localhost` was getting passed in. I need clarification on the purpose of `build-and-run.sh` to figure out how.